src/firefly/js/templates/fireflyviewer/TriViewPanel.jsx

```diff
@@ -104,10 +105,10 @@ function RightSide({expanded, closeable, showXyPlots, showMeta, showFits, dataPr
         } else {
             return makeActiveChartTab({viewerId, activeLabel, chartExpandedMode, closeable});
         }
-    };
+    }
 
     const style= {height: '100%'};
-    const defaultSelected= coverageRight ? 'coverage' : showXyPlots ? 'activeCharts' : 'fits';
+    const defaultSelected= true ? 'rowDetails' : coverageRight ? 'coverage' : showXyPlots ? 'activeCharts' : 'fits'; //TODO: remove after testing
     const key= `${showXyPlots&&'xyplot'}-${cov&&'cov'}-${meta&&'meta'}-${fits&&'fits'}`;
```

Comment:
I don't see where it says 'rowDetails' should be the default selected tab.

Reply:
This was for debugging

Reply:
See "TODO: remove after testing"